UI/ChatUI/ChatUI/ChatUI/Models/MessageModel.cs

```diff
@@ -0,0 +1,26 @@
+using System.Threading;
+using System.Threading.Tasks;
+using ChatUI.Entities;
+using ChatUI.Services;
+using Uno.Extensions.Reactive;
+
+namespace ChatUI.Models;
+
+public partial record MessageModel(IMessageService MessageService)
+{
+	public IListState<Message> Messages => ListState
+		.Async(this, MessageService.GetMessages);
+
+	public IState<Message> NewMessage => State<Message>.Value(this, () => Message.Empty);
+
+	public async ValueTask AddMessage(CancellationToken ct)
+	{
+		var newMessage = await NewMessage;
```

Comment:
```suggestion
	public async ValueTask AddMessage(Message newMessage, CancellationToken ct)
	{
```

You can use the implicit feed parameter injection. Note that this will also disable the command as long as the `NewMessage` is _empty_.

Reply:
Fixed.